# values in the equation for a gaussuan distribution
float64 amplitude

# location and variable informaiton
float64 offset_x
float64 offset_y

# ellipsis information
contour_node/ellipse ellipse